### PR TITLE
feature(data): add fiche sp fetcher

### PR DESCRIPTION
**Pull request by UnbearableBear**
Any thoughts on that ?

### CODE REVIEW

File: packages/code-du-travail-data/dataset/fiches_service_public/format.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const parseReference = require("./parseReference");
+const parseReference = require("./parse-reference");
```
Comment (lionelB) [inline packages/code-du-travail-data/dataset/fiches_service_public/format.js]: why did you rename file here ? most of the filename are camelCase

**Review by lionelB — COMMENTED** [review]
Don't see why you revert the filenames to camelCase since most of script are camelCase here

**Comment by lionelB** [comment]
I wonder if using bash would be a better in that it's not tied to node env ? 
but  👍  !

**Comment by UnbearableBear** [comment]
Maybe in a future version (i'm not very familiar with bash processes) but all the parsing is based on node right now so I guess it's ok

File: packages/code-du-travail-data/dataset/fiches_service_public/format.js (hunk 1 above)
Comment (UnbearableBear) [reply packages/code-du-travail-data/dataset/fiches_service_public/format.js]: Consistency, some files were camelCased some others not so I picked the most common (in this folder). I always prefer files not to be camel cased though. Old habitude, I find it slighty more readable, and by convention "-" means "these are two distinct words".

**Comment by lionelB** [comment]
@UnbearableBear est ce que tu dé-renomer les fichiers ? le kebab-case c'est utilisé pour les fichier json les scripts sont en camelCase

**Comment by UnbearableBear** [comment]
@lionelB je m'en occupe

**Review by lionelB — APPROVED** [review]
